makefile: unknown command python3

I've loved being able to use the makefile + docker-compose setup you've had here since last month - but I just tried pulling a totally fresh copy of the repo and running make, and got the error

```
No Django settings specified.
Unknown command: 'python3'
Type 'manage.py help' for usage.
make: *** [janeway] Error 1
```

can't fathom why the container has forgotten where python is? Any thoughts?

--- a/Makefile
+++ b/Makefile
@@ -48,7 +48,7 @@ help:		## Show this help.
 janeway:	## Run Janeway web server in attached mode. If NO_DEPS is not set, runs all dependant services detached.
 	docker-compose $(_VERBOSE) run $(NO_DEPS) --rm --service-ports  janeway-web $(entrypoint)
 command:	## Run Janeway in a container and pass through a django command passed as the CMD environment variable
-	docker-compose run $(NO_DEPS) --rm janeway-web src/manage.py $(CMD)
+	docker-compose run $(NO_DEPS) --rm janeway-web $(CMD)
 install:	## Run the install_janeway command inside a container
 	touch db/janeway.sqlite3
 	bash -c "make command CMD=install_janeway"

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -33,7 +33,6 @@ services:
 
   janeway-web:
     build: .
-    command: python3 src/manage.py runserver 0.0.0.0:8000
     volumes:
       - .:/vol/janeway
     ports:

